src/main/java/com/vaadin/training/bugrap/presenter/MainView.java

```diff
@@ -26,9 +28,24 @@ public void enter(final ViewChangeEvent event) {
 			getUI().getNavigator().navigateTo(Views.LOGIN);
 		});
 
+		initProjectSelect();
+
+		initSearchReportsDropdown();
+	}
+
+	private void initProjectSelect() {
 		final List<Project> projects = new ArrayList<>(BugrapFacade.getInstance().findProjects());
 		Collections.sort(projects);
 		projectSelect.setItems(projects);
 	}
 
+	private void initSearchReportsDropdown() {
+		final SimpleSearchDropDown searchReportsDropDown = new SimpleSearchDropDown(query -> {
+			return Collections.emptyList();
```

Comment:
This might be a good place to add a TODO-marker so the implementation is not forgotten

Reply:
Ok!